### ISSUE TITLE
Started another set of tracking/Rstar/tauI runs

### ISSUE THREAD

**Issue by donahuem** [lizzieinvancouver/temporalvar#18]
8/30/2018 12:26 pm PDT
started another set of runs with all four types
Array IDs are:  52031833, 52031904, 52031950, 52031996
Each array has 50 tasks, each task completes 100 runs
All output in /wolkovich_lab/temporalvar/R/output/[Bout, SummaryFiles, or OtherOut/ArrayID

**Comment by lizzieinvancouver** [comment]
@donahuem Can you check on 52031833 I didn't see it and 52031904 seems small.

**Comment by lizzieinvancouver** [comment]
changed SummaryOut_52031904-2.txt to SummaryOut_52031904-1.txt

**Comment by donahuem** [comment]
52031833:  sacct says it completed, but there is no record of it.  Perhaps there was an error in the write process from scratch?
52031904: sacct says all runs complete but only 6 of the runs wrote to R/output/SummaryOut.  Again, I have to assume that these runs ran out of scratch space.  Unfortunately, either the error files didn't write or I deleted them, so I can't check.
52031950:  sacct says all runs complete, and 44/50 runs wrote to SummaryOut.  
52031996:  same but only 6 written to SummaryOut.

Closing this issue.  Will create new runs
